docs/tutorials/noisy_qsimcirq.ipynb

```diff
@@ -0,0 +1,290 @@
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "[0.52631575+0.j 0.49930704+0.j 0.49930704+0.j 0.47368425+0.j]\n"
+     ]
+    }
+   ],
+   "source": [
+    "qsim_simulator = qsimcirq.QSimSimulator()\n",
+    "results = qsim_simulator.simulate(circuit)\n",
+    "print(results.final_state_vector)"
+   ]
+  },
+  {
+   "source": [
+    "It's important to note that unlike density-matrix simulations, this result is stochastic in nature. Running the circuit multiple times may yield different results, but each result generated is a possible outcome of the provided circuit."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## Other simulation modes\n",
+    "\n",
+    "Noisy circuit simulation in qsimcirq supports all of the same simulation modes as the noiseless simulator, including:\n",
+    "\n",
+    "### Measurement Sampling"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "# Simulate measuring at the end of the circuit.\n",
+    "measured_circuit = circuit + cirq.measure(q0, q1, key='m')\n",
+    "measure_results = qsim_simulator.run(measured_circuit, repetitions=5)\n",
+    "print(measure_results)"
+   ],
+   "cell_type": "code",
+   "metadata": {},
+   "execution_count": 21,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "1,0=10101, 00000\n"
+     ]
+    }
+   ]
+  },
+  {
+   "source": [
+    "### Amplitude evaluation"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "[0j, (0.8164965510368347+0j)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Calculate only the amplitudes of the |00) and |01) states.\n",
+    "amp_results = qsim_simulator.compute_amplitudes(\n",
+    "    circuit, bitstrings=[0b00, 0b01])\n",
+    "print(amp_results)"
+   ]
+  },
+  {
+   "source": [
+    "### Expectation values\n",
+    "\n",
+    "Special attention is warranted for this use case. In the other modes discussed above, the stochastic nature of the noisy simulator's results did not affect the validity of the results; a sample from this simulator is a valid representation of a possible sample from executing the circuit on hardware with similar noise.\n",
+    "\n",
+    "Expectation values, as a measure of the average value of an observable, cannot be extracted so easily. An expectation values can only be calculated from trajectories by repeated simulation - and even then, the results are only an estimate of the true expectation values. This is to be expected, as simulating all possible trajectories is infeasible for all but the simplest circuits.\n",
```

Comment:
I would change this to:
    "Expectation values an be estimated from trajectories by repeated simulation. This is demonstrated below."

Removing:
"Special attention is warranted for this use case. In the other modes discussed above, the stochastic nature of the noisy simulator's results did not affect the validity of the results; a sample from this simulator is a valid representation of a possible sample from executing the circuit on hardware with similar noise.\n",
    "\n",
    "Expectation values, as a measure of the average value of an observable, cannot be extracted so easily. An expectation values can only be calculated from trajectories by repeated simulation - and even then, the results are only an estimate of the true expectation values. This is to be expected, as simulating all possible trajectories is infeasible for all but the simplest circuits.\n"
    "A demonstration of this can be seen below."

Reply:
I think it's important to highlight that (1) expectation values from this method are estimates, since we don't have the full density matrix, and (2) adding repetitions increases the accuracy of this method. To capture those details, I propose:

"Expectation values can only be estimated from trajectories, but the accuracy of this estimates can be increased by simulating the circuit additional times. This is demonstrated below."

What do you think?

Reply:
SGTM (typo: this estimates -> these estimates)

Reply:
Done.